apps/backend/src/core/logging.py

```
@@ -0,0 +1,87 @@
+def _add_request_id_event_dict(
+    logger: Any, method_name: str, event_dict: MutableMapping[str, Any]
+) -> MutableMapping[str, Any]:
+    """Structlog processor to inject request_id from contextvars."""
+    rid = request_id_var.get()
+    if rid is not None:
+        event_dict["request_id"] = rid
+    return event_dict
+
+
+def _iso_time(_: Any, __: str, event_dict: MutableMapping[str, Any]) -> MutableMapping[str, Any]:
+    event_dict["timestamp"] = datetime.now(timezone.utc).isoformat()
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)